guides/source/association_basics.md

```diff
@@ -2724,3 +2724,156 @@ will run a query like:
 ```sql
 SELECT "vehicles".* FROM "vehicles" WHERE "vehicles"."type" IN ('Car')
 ```
+
+Delegated Types
+----------------
+
+[`Single Table Inheritance (STI)`](#single-table-inheritance-sti) works best when there is little difference between subclasses and their attributes, but includes all attributes of all subclasses I need to create a single table.
+
+This has the disadvantage that attributes that are only used by a specific subclass are added to the same table, even though they are unnecessary attributes for another subclass, which bloats a single table.
```

Comment:
This seems confusing to me, maybe too wordy? Let me try.

```suggestion
The disadvantage of this approach is that it results in bloat to that table. Since it will even include attributes specific to a subclass that aren't used by anything else.
```

Reply:
thank you. I replaced it with Suggest.

https://github.com/rails/rails/pull/46190/files#diff-d977574b785ce95fe656c58211c668992bf95214d52d3ffadb409ef8f69ad287R2733